cookbooks/swift/templates/default/etc/swift/proxy-server/proxy-server.conf.d/20_settings.conf.erb

```diff
@@ -6,7 +6,7 @@ log_statsd_port = 9120
 <% end -%>
 
 [pipeline:main]
-pipeline = catch_errors healthcheck proxy-logging domain_remap cache <%= @nvratelimit_pipeline %> etag-quoter container_sync bulk tempurl s3api tempauth ratelimit slo dlo versioned_writes symlink <%= @keymaster_pipeline %> encryption subrequest-logging proxy-server
+pipeline = catch_errors healthcheck proxy-logging domain_remap cache <%= @nvratelimit_pipeline %> etag-quoter container_sync bulk tempurl s3api <%= @nvauth_pipeline %> tempauth ratelimit slo dlo versioned_writes symlink <%= @keymaster_pipeline %> encryption subrequest-logging proxy-server
```

Comment:
I really don't want to figure out how to make this plugable in ruby/vsaio, having an interface upstream that middlewares can use to describe their pipeline requirements in terms of "before" and "after" with `pipeline = auto` support in `swift.common.wsgi` would be better.